app/routes/contact/components/ContactForm.tsx

```diff
@@ -98,10 +97,23 @@ const ContactForm = (props: Props) => {
       />
 
       <p>
-        Du kan velge å sende meldingen med anonym avsender. De som mottar
-        meldingen vil ikke få vite hvem som har opprettet meldingen. De vil da
-        heller ikke ha mulighet til å svare på meldingen. Ønsker om innsyn kan
-        ikke sendes inn anonymt.
+        Ved anonym innsendelse vil de som mottar meldingen ikke få vite hvem som
+        har opprettet den, men de vil da heller ikke ha mulighet til å svare. Av
+        den grunn kan ikke ønsker om innsyn sendes inn anonymt.
+      </p>
+      <p>
+        For mer sensitive saker eller hvis du ønsker å varsle anonymt med
+        mulighet for respons, burde du bruke vår{' '}
+        <a
+          href="https://avvik.abakus.no"
+          rel="noopener noreferrer"
+          target="_blank"
+        >
+          tredjeparts varslingsportal
+        </a>
+        . Dette sikrer i større grad at du kan kommunisere din bekymring uten å
+        avsløre din identitet, samtidig som det gir oss mulighet til å håndtere
+        situasjonen på en effektiv måte.
```

Comment:
I would like this to be at the top and be clear _before_ the user starts filling out the form.

Reply:
Sure, but at the same time I think having the text here makes the checkbox right below more visible. It's sort of hidden by all the text and input fields and I'm not sure how to make it pop out more.

Reply:
Moved it to the top now.